jekyll/_cci2/storage-lifecycle.adoc

```diff
@@ -0,0 +1,82 @@
+= CircleCI Server AWS S3 Storage Lifecycle
+:page-layout: classic-docs
+:page-liquid:
+:source-highlighter: rouge
+:rouge-linenums-mode: inline
+:icons: font
+:toc: macro
+:toc-title:
+:sectanchors:
+
+
+[.serveronly]_This document is intended for system administrators of self-hosted installations of CircleCI Server._
+
+This guide is intended to help system administrators of CircleCI Server installations on AWS better understand how S3 storage is used. This can help to cut compute resource costs and avoid issues for service users when removing resources.
+
```

Comment:
Could you add a emphasized note around here that, at least for 2.19.x, the lifecycle policy below is _not_ configured out-of-box, and is needed to be manually configured?
Actually there are multiple customers who believe that there is a built-in functional garbage-collecting mechanism, for whom we act like MythBusters to tell them that there is not.